### DISCUSSION TITLE
Appreciation for traditional desktops and GUI design

### DISCUSSION THREAD

**Discussion by contrarybaton60** [ivan-hc/AM#1256] [General]
In recent years we all see how most popular software went away from menubars and to more "dumb-down" more mobile style looks and it kinda hurts to see that.

A lot of users see that change and went away from it.

Cinnamon, MATE, Xfce, Budgie... heck, even Unity is more considered "traditional" than the modern GNOME and KDE.

On a individual app basis... we're still fine, it's hard to get something like a IDE/code editor or a office program to be "dumbed-down" and most applications still use them (Virtualbox, VLC, Firefox and Thunderbird, Evolution, pretty much all Xfce, Mate, and Linux Mint-specfic programs...)

This is just an appreciation post for this old yet still powerful UI design, bye.

**Comment by contrarybaton60** [comment]
mmm global menu
![image](https://github.com/user-attachments/assets/dd93dbf3-6ed9-4036-bce6-5df6d301e73c)

**Reply by Samueru-sama** [reply]
how did you get that global menu? iirc the xfce4-panel had a plugin for it but it was abandoned

**Reply by contrarybaton60** [reply]
I used a Ubuntu Unity vm for the screenshot, Unity's global menu is much better than MATE global menu plugin in my tests

**Reply by contrarybaton60** [reply]
AppImages are a bit flaky when it comes to global menus, VSCodium and ivan's builds of VLC have support and it's quite good, but neither ivan's builds, the flatpak or snap version of GIMP have support, only the .deb from Ubuntu's repos.
![image](https://github.com/user-attachments/assets/d12583cc-e43e-4a9c-b998-89ea84aa2042)
![image](https://github.com/user-attachments/assets/f460724c-26a2-4608-870e-9006d2013d2b)

**Reply by contrarybaton60** [reply]
Unity (or KDE with that one plugin that I can't remember the name of) likely have the best global menu support, but support is flaky if you get your applications other than the native Ubuntu repos

**Reply by Samueru-sama** [reply]
Try this GIMP AppImage: https://github.com/pkgforge-dev/GIMP-AppImage

**Reply by contrarybaton60** [reply]
https://github.com/user-attachments/assets/cdffdbd2-9b03-4cb0-8950-7170a3d8d096

nope, doesn't work

**Reply by Samueru-sama** [reply]
Looks like all global menus used a modified version of [gtk](https://gitlab.com/vala-panel-project/vala-panel-appmenu/) that adds the global menu functionality.

In other words the gtk libraries in the AppImage have to come with the feature.

**Reply by contrarybaton60** [reply]
Ubuntu is the only distro I know of that patches gtk apps to have them, which is why the global menu works there and not on Debian and others

**Reply by contrarybaton60** [reply]
Debian does contain `vala-appmenu` packages but it's kinda borked

**Comment by Samueru-sama** [comment]
>it's hard to get something like a IDE/code editor or a office program to be "dumbed-down" and most applications still use them

https://github.com/lite-xl/lite-xl

**Comment by contrarybaton60** [comment]
![Linux Mint 22.1 beta with the Mint-L-Purple theme](https://github.com/user-attachments/assets/660240a7-ac3d-4c78-8eb9-6b2fb99b8468)
![Ubuntu Unity 24.04 LTS with the default Yaru theme](https://github.com/user-attachments/assets/fb8398cc-fdb4-408e-a381-33dbaa4a6fb1)
![Fedora 41 MATE+Compiz with Clearlooks/TraditionalOK theme](https://github.com/user-attachments/assets/e71390f0-b796-474b-9b6f-681134617e1a)
![screenshot of Xubuntu because I haven't been bothered to install a vm with xfce](https://github.com/user-attachments/assets/ee0292b8-981b-481e-8ca6-b39477587a72)
screenshots
